Could you check if that is related to the PR?

```
FAIL: //tensorflow/core:lib_monitoring_sampler_test (see /tmpfs/tmp/bazel/execroot/org_tensorflow/bazel-out/k8-opt/testlogs/tensorflow/core/lib_monitoring_sampler_test/test.log)
INFO: From Testing //tensorflow/core:lib_monitoring_sampler_test:
Traceback (most recent call last):
  File ".asci-reserved/run_docker.py", line 568, in <module>
    main()
  File ".asci-reserved/run_docker.py", line 544, in main
    cmd = setup_and_generate_command(sys.argv[1:])
  File ".asci-reserved/run_docker.py", line 145, in setup_and_generate_command
    pull_spec_if_necessary(img_spec)
  File ".asci-reserved/run_docker.py", line 206, in pull_spec_if_necessary
    if has_spec(img_spec):
  File ".asci-reserved/run_docker.py", line 234, in has_spec
    return spec.digest in listing.split()
AttributeError: 'NoneType' object has no attribute 'split'
```

--- a/tensorflow/contrib/learn/python/learn/estimators/model_fn.py
+++ b/tensorflow/contrib/learn/python/learn/estimators/model_fn.py
@@ -30,8 +30,8 @@ from tensorflow.contrib.learn.python.learn.estimators import prediction_key
 from tensorflow.python.estimator import model_fn as core_model_fn_lib
 from tensorflow.python.estimator.export import export_output as core_export_lib
 from tensorflow.python.framework import dtypes
-from tensorflow.python.framework import framework_lib
 from tensorflow.python.framework import ops
+from tensorflow.python.framework import sparse_tensor
 from tensorflow.python.framework import tensor_shape
 from tensorflow.python.ops import array_ops
 from tensorflow.python.platform import tf_logging as logging
@@ -156,11 +156,11 @@ class ModelFnOps(
     else:
       if isinstance(predictions, dict):
         predictions = {
-            k: framework_lib.convert_to_tensor_or_sparse_tensor(v)
+            k: sparse_tensor.convert_to_tensor_or_sparse_tensor(v)
             for k, v in six.iteritems(predictions)
         }
       else:
-        predictions = framework_lib.convert_to_tensor_or_sparse_tensor(
+        predictions = sparse_tensor.convert_to_tensor_or_sparse_tensor(
             predictions)
 
     # Validate eval_metric_ops

